tools/docker/Dockerfile

```diff
@@ -151,7 +151,7 @@ COPY _submodules/firecracker/target/$FIRECRACKER_TARGET/release/firecracker /usr
 COPY _submodules/firecracker/target/$FIRECRACKER_TARGET/release/jailer /usr/local/bin/
 COPY _submodules/runc/runc /usr/local/bin
 COPY tools/image-builder/rootfs.img /var/lib/firecracker-containerd/runtime/default-rootfs.img
-COPY runtime/firecracker-runc-config.json.example /etc/containerd/firecracker-runc-config.json
+COPY tools/docker//firecracker-runc-config.json /etc/containerd/firecracker-runc-config.json
```

Comment:
did we remove the other file at `runtime/firecracker-runc-config.json.example`?  I think we should only have one for simplicity

Reply:
Done